composer.json

```diff
@@ -33,7 +33,7 @@
     "require": {
     },
     "require-dev": {
-        "cakephp/cakephp": "~3.0",
+        "cakephp/cakephp": "dev-master",
```

Comment:
This is temporarily for cakephp/cakephp#7491, this is purely to get the tests to run. This can be reverted once a CakePHP version has been released with that included.